mttl/models/containers/__init__.py

```diff
@@ -165,21 +147,22 @@ def replace_selector_for_container(
                         supports_config
                     )
                     # selector does not apply to this container
-                    if not container_modifier == modifier_type:
+                    if not container_modifier == modifier_name:
                         continue
                     else:
                         expert_containers.append(layer)
                         break
 
     if not expert_containers:
-        raise ValueError(
-            f"No expert containers found for modifier type: {modifier_type}. Cannot assign a selector! Load some experts beforehand."
+        logger.warn(
```

Comment:
could you use logger.warning? `warn` will eventually be deprecated (deprecationWarning)

Reply:
actually reinstated value error